src/main/java/data/Step.java

```diff
@@ -0,0 +1,18 @@
+public class Step {
+
+  private String direction;
```

Comment:
Maybe change to "description" for clarity? I got confused during the first pass (i.e. thought direction was NESW) but maybe that's just me.

Reply:
I changed it to instruction